feature-libs/product/configurators/common/src/components/service/config-ui-key-generator.service.ts

```diff
@@ -0,0 +1,122 @@
+import { Configurator } from '@spartacus/core';
```

Comment:
As this is now just a class with static methods, let's drop the _service_ from the name. 
Maybe `ConfigUIKeyGeneratorUtil`?

Reply:
Done. I just called it ConfigUIKeyGenerator, in my opinion there is no need to call it "util" in the end.